app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -180,7 +180,7 @@ def with_version4_vm_service(vm)
   end
 
   def use_ovirt_sdk?
-    ::Settings.ems.ems_redhat.use_ovirt_engine_sdk
+    true
```

Comment:
Why is this method needed if `use_ovirt_sdk?` always return true ?
Is this being overridden by any flow ?

Reply:
No we do not need it at all, I will remove this at a later refactoring, together with lots of other code, see: https://github.com/ManageIQ/manageiq-providers-ovirt/pull/388#discussion_r304888050

In this case there are other pieces of code like builders (parser builder, event builder) to be removed later as well.